docs/calico-with-docker/docker-network-plugin/CalicoAsService.md

```diff
@@ -0,0 +1,86 @@
+<!--- master only -->
+> ![warning](../../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Running Calico Node Containers as a Service
+
+The `calico/node` and `calico/node-libnetwork` containers can be run as individual
```

Comment:
This still feels like an odd intro to me.

How about something like:

"This guide explains how to run Calico as a system process or service.  We include examples for Systemd, but the details can be applied to other init daemons such as upstart".

That way the libnetwork section becomes an additional section where all you need to say is "If you are running Calico as a Docker network plugin, it is also necessary to have a separate job for handling the Calico libnetwork driver".